### PR TITLE
Show training status of data collector 

### PR DESCRIPTION
**Pull request by VihoSerge**
**PR Training status**

```
    Summary of the changes
    - Show training status
    - Manually set training status

    Fixes #756
```

### CODE REVIEW

File: Source/Reporting/Web/src/app/Management/DataCollectors/ChangeBaseInformation.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ export class ChangeBaseInformation extends Command
         this.displayName = '';
         this.yearOfBirth = 0;
         this.sex = {};
+        this.status = 1;
```
Comment (woksin) [inline Source/Reporting/Web/src/app/Management/DataCollectors/ChangeBaseInformation.js]: There should be a command doing this, I can do that :)

File: Source/Reporting/Web/src/app/Management/DataCollectors/edit/edit.html [hunk 2]
```diff
@@ -148,5 +148,22 @@ <h1>Edit Data Collector</h1>
             </div>
         </div>
     </div>
+    <div class="form-group">
+        <label for="status">Status</label>
+        <select class="form-control" id="status"
+                required
+                [(ngModel)]="changeBaseInformationCommand.status"
```
Comment (woksin) [inline Source/Reporting/Web/src/app/Management/DataCollectors/edit/edit.html]: There should be a seperate command for setting a data collector in / out of training. I can do that :)

File: Source/Reporting/Web/src/app/Management/DataCollectors/edit/edit.ts [hunk 3]
```diff
@@ -125,6 +125,7 @@ export class Edit implements OnInit {
         this.changeBaseInformationCommand.displayName = this.dataCollector.displayName;
         this.changeBaseInformationCommand.fullName = this.dataCollector.fullName;
         this.changeBaseInformationCommand.sex = this.dataCollector.sex;
+        this.changeBaseInformationCommand.status = this.dataCollector.inTraining ? 1 : 0;
```
Comment (woksin) [inline Source/Reporting/Web/src/app/Management/DataCollectors/edit/edit.ts]: There should be a command for setting a data collector in / out of training

File: Source/Reporting/Web/src/app/Management/DataCollectors/list/list.html [hunk 4]
```diff
@@ -18,7 +18,7 @@ <h2>Data collectors</h2>
             <th>Full Name</th>
             <th>Display Name</th>
             <th>Phone Numbers</th>
-            <th></th>
+            <th>Status</th>
```
Comment (woksin) [inline Source/Reporting/Web/src/app/Management/DataCollectors/list/list.html]: I think that 'InTraining' is a better name than 'Status', it is more explicit

**Comment by cathinkaw** [comment]
Fixed https://github.com/IFRCGo/cbs/issues/756. Sindre can maybe add his changes to the backend to this branch before we merge it into the master (his 2 comments). But all the frontend changes look good 👍

**Review by woksin — APPROVED** [review]
Good work! 👍

**Pull request merged**